api/bus.go

```diff
@@ -593,61 +593,42 @@ type (
 	}
 
 	ContractSetMetricsQueryOpts struct {
-		After  time.Time
-		Before time.Time
-
 		Name string
-
-		Offset int
-		Limit  int
 	}
 
 	ContractSetChurnMetric struct {
-		Direction string               `json:"direction"`
-		FCID      types.FileContractID `json:"fcid"`
-		Name      string               `json:"name"`
-		Reason    string               `json:"reason,omitempty"`
-		Timestamp time.Time            `json:"timestamp"`
+		Direction  string               `json:"direction"`
+		ContractID types.FileContractID `json:"contractID"`
+		Name       string               `json:"name"`
+		Reason     string               `json:"reason,omitempty"`
+		Timestamp  time.Time            `json:"timestamp"`
 	}
 
 	ContractSetChurnMetricsQueryOpts struct {
-		After  time.Time
-		Before time.Time
-
 		Name      string
 		Direction string
 		Reason    string
-
-		Offset int
-		Limit  int
 	}
 
 	PerformanceMetric struct {
 		Action    string          `json:"action"`
-		Host      types.PublicKey `json:"host"`
+		HostKey   types.PublicKey `json:"host"`
```

Comment:
update json field name

Reply:
actually searching for it shows another inconsistency in our `Sector` type

Reply:
I think there is another one too but I'd rather avoid updating existing tags in this PR and breaking the UI by accident. We can do that separately.